assets/python/ml/pyml:data_input:train_test_split:sklearn.j2.py

```diff
@@ -0,0 +1,38 @@
+# ----------------------------------------------------------------- #
+#                                                                   #
+#   Workflow Unit to perform a train/test split                     #
+#                                                                   #
+# ----------------------------------------------------------------- #
+
+import sklearn.model_selection
+import numpy as np
+import settings
+
+with settings.context as context:
+    # Train
+    if settings.is_workflow_running_to_train:
+        # Load training data
+        train_target = context.load("train_target")
+        train_descriptors = context.load("train_descriptors")
+
+        # Combine datasets to facilitate train/test split
+
+        # Do train/test split
+        # percent_held_as_test is the amount of the dataset held out as the testing set. If it is set to 0.2,
```

Comment:
Let's include variable name in quotes "`"  for percent_held_as_test?

Reply:
Comment has been changed to include the variable name in quotes